### PR TITLE
#158254668 Add help text to import script

### PR DESCRIPTION
**Pull request by JaphethCA**
## What does this PR do?
- It adds help text to import script

## Description of Task to be completed
- adds help text support with `argparse`

## Relevant Pivotal Tracker stories
[#158254668](https://www.pivotaltracker.com/story/show/158254668)

## Screenshots
screen shot 2018-06-28 at 10 01 18 am

### CODE REVIEW

File: utils/seed_asset.py [hunk 1]
```diff
@@ -22,6 +22,14 @@
...
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        prog="seed_asset",
+        description="""
+            The program for importing of assets
+        """
+    )
+    args = parser.parse_args()
+
```
Comment (egichuri) [inline utils/seed_asset.py]: @JaphethCA `if not show the message and continue with execution.` - is this possible?

File: utils/seed_asset.py [hunk 2]
```diff
@@ -22,6 +22,14 @@
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        prog="seed_asset",
+        description="""
+            The program for importing of assets
```
Comment (egichuri) [inline utils/seed_asset.py]: we should include the expected column names here.

Comment (egichuri) [reply utils/seed_asset.py]: and file format, etc

File: utils/seed_asset.py (hunk 1 above)
Comment (JaphethCA) [reply utils/seed_asset.py]: Am sorry, but I don't understand what you mean. Can you please give more details.

File: utils/seed_asset.py (hunk 2 above)
Comment (JaphethCA) [reply utils/seed_asset.py]: Okay, Will do that. Thanks

File: utils/seed_asset.py (hunk 1 above)
Comment (egichuri) [reply utils/seed_asset.py]: currently the `-h` argument shows the help text and exits.. without the `-h` it goes straight to execution (asks user to enter the csv file name) without displaying the help text. is it possible to show the help text even without the `-h`?

Comment (JaphethCA) [reply utils/seed_asset.py]: Sure, but how will user run the command to input csv file or link if running the script without arguments return a help text?

**Pull request merged**